index.html

```diff
@@ -36,6 +36,321 @@ <h5>LET’S CONNECT</h5>
              </ul>
             </div>
         </section>
+        <section class="Snapshoot">
+
+
+          <article class="article1">
+           <figure>
+            <img src="images/Snapshoot Portfolio.png" alt="Portfolio1">
+           </figure>
+
+
+              <div class="article1div">
+
+
+                  <h3 class="texttitle font1">Tonic</h3>
+
+
+                  <ul>
+
+
+                    <li class="texttitle font2">CANOPY</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">Back End Dev</li>
+                    <li class="texttitle font3"><img src="images/counter.png"></li>
+                    <li class="texttitle font3">2015</li>
+
+
+                  </ul>
```

Comment:
- Kindly, be sure that the bullet icons are visible 👍👌 and it is best to use style instead of image to display bullets👌👍